src/category_theory/limits/yoneda.lean

```diff
@@ -53,4 +55,73 @@ colimit.iso_colimit_cocone { cocone := _, is_colimit := colimit_cocone_is_colimi
 
 end coyoneda
 
+variables {C : Type u} [category.{v} C]
+
+open limits
+
+/-- The yoneda embedding `yoneda.obj X : Cᵒᵖ ⥤ Type v` for `X : C` preserves limits. -/
+instance yoneda_preserves_limits (X : C) : preserves_limits (yoneda.obj X) :=
+{ preserves_limits_of_shape := λ J 𝒥, by exactI
+  { preserves_limit := λ K,
+    { preserves := λ c t,
+      { lift := λ s x, has_hom.hom.unop (t.lift ⟨op X, λ j, (s.π.app j x).op, λ j₁ j₂ α, _⟩),
+        fac' := λ s j, funext $ λ x, has_hom.hom.op_inj (t.fac _ _),
+        uniq' := λ s m w, funext $ λ x,
+        begin
+          refine has_hom.hom.op_inj (t.uniq ⟨op X, _, _⟩ _ (λ j, _)),
+          { dsimp, simp [← s.w α] }, -- See library note [dsimp, simp]
+          { exact has_hom.hom.unop_inj (congr_fun (w j) x) },
+        end }
+        } } }
```

Comment:
```suggestion
        end } } } }
```
and similarly on line 89?

Reply:
I already made this change in the parent PR so when that merges I think it should automatically be changed here, thanks!